301-spinnaker-vmss-or-aks/azuredeploy.json

```diff
@@ -0,0 +1,402 @@
+      "allowedValues": [
+        "sshPublicKey",
+        "password"
+      ],
+      "metadata": {
+        "description": "Type of authentication to use on the Virtual Machine. SSH key is recommended."
+      }
+    },
+    "sshPublicKey": {
+      "defaultValue": "",
+      "type": "string",
+      "metadata": {
+        "description": "Configure the linux machine or the Jenkins instance with the SSH public key string. Your key should include three parts, for example'ssh-rsa AAAAB...snip...UcyupgH azureuser@linuxvm'"
+      }
+    },
+    "adminPassword": {
+      "defaultValue": "",
+      "type": "securestring",
+      "metadata": {
+        "description": "Password for the Virtual Machine or the Jenkins instance."
+      }
+    },
+    "servicePrincipalAppId": {
+      "type": "string",
+      "metadata": {
+        "description": "Service Principal App ID (also called Client ID) that has contributor rights to the subscription used for this deployment. It is used by Spinnaker to dynamically manage resources."
+      }
+    },
+    "servicePrincipalAppKey": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Service Principal App Key (also called Client Secret) that has contributor rights to the subscription used for this deployment. It is used by Spinnaker to dynamically manage resources."
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    },
+    "aksClusterName": {
+      "defaultValue": "",
+      "type": "string",
+      "metadata": {
+        "description": "AKS cluster name"
+      }
+    },
+    "aksResourceGroup": {
+      "defaultValue": "",
+      "type": "string",
+      "metadata": {
+        "description": "AKS resource group"
+      }
+    }
+  },
+  "variables": {
+    "scenarioPrefix": "devops",
+    "storageAccountName": "[concat(variables('scenarioPrefix'), uniqueString(resourceGroup().id))]",
+    "vmStorageAccountContainerName": "vhds",
+    "keyVaultName": "[concat('vault', uniqueString(resourceGroup().id))]",
+    "vnetName": "[concat(variables('scenarioPrefix'), 'Vnet')]",
+    "subnet0Name": "[concat(variables('scenarioPrefix'), 'Subnet0')]",
+    "subnet1Name": "[concat(variables('scenarioPrefix'), 'Subnet1')]",
+    "subnet2Name": "[concat(variables('scenarioPrefix'), 'Subnet2')]",
+    "frontEndNSGName": "[concat(variables('scenarioPrefix'), 'NSG')]",
+    "nicName": "[concat(variables('scenarioPrefix'), 'Nic')]",
+    "publicIPAddressName": "[concat(variables('scenarioPrefix'), 'PublicIP')]",
+    "vmName": "[concat(variables('scenarioPrefix'), 'VM')]",
+    "vmExtensionName": "[concat(variables('scenarioPrefix'), 'Init')]",
+    "_artifactsLocation": "https://raw.githubusercontent.com/Azure/azure-devops-utils/master/",
+    "_extensionScript": "101-spinnaker-install-selection.sh",
+    "_artifactsLocationSasToken": "",
```

Comment:
remove this since it's never used

Reply:
> remove this since it's never used

It is used at line 373 and 377.

Reply:
If here it means "_artifactsLocationSasToken" variable, I've removed it.